app/src/assets/scss/custom/base/_common.scss

```diff
@@ -77,4 +77,25 @@
 .es-product-background {
   background: url("../../../img/vegetable.png") no-repeat center center;
   background-size: cover;
-}
\ No newline at end of file
+}
+
+.bw-bg {
+  background-image: linear-gradient(to bottom, white 50%, black 50%);
+}
+
+.es-fs-1 {
+  font-size: 1rem;
+}
+
+.es-fs-07 {
+  font-size: 0.7rem;
+}
+
+.es-fs-13 {
+  font-size: 1.3rem;
+}
+
+.es-size-box {
+  width: 45px;
+  height: 45px;
+}
```

Comment:
nazwa tej klasy nie mówi nam nic co to za wielkość i do czego

Reply:
ok

Reply:
I changed className on  .es-size-box-45px